### ISSUE TITLE
Import of empty repository from Github is very slow (possibly not working)

### ISSUE THREAD

**Issue by iainsproat** [GitbookIO/feedback#499]
## Category

Bug on Gitbook.com and integration with github

## What is the current behavior?

1. Create an empty repository on Github.com
2. Create a new private book by importing from Github.com; select the empty repository
3. Click Create Book
4. *Wait some time* (I'm about 2 or 3 hours currently)
5. Go the new book welcome page, and it says "Your content is being imported"

Also
5. Go to the dashboard, click edit on the new book
6. the spinners are shown but nothing loads.

## What is the desired behavior?

I'd expect within a minute or two at most to have the empty repository imported from Github and be able to edit the book

## Additional information

@iainsproat

**Comment by szkrd** [comment]
Nope, it's not working for sure. To me setting up a new repo and linking it to gitbook seems like a logical workflow, yet we are stuck at the "your book is being imported" and no errors are shown :(
